test/lib/bamboo/adapters/test_adapter_test.exs

```diff
@@ -157,4 +159,16 @@ defmodule Bamboo.TestAdapterTest do
 
     assert_delivered_email sent_email
   end
+
+  # The actual sent email and the email tested may have differences in their assigns
+  # Even if the resulting body is the same
+  test "assertion helpers remove assigns" do
```

Comment:
I think the comment can be removed and maybe the test can say something like `"delivered emails have normalized assigns"` or something like that. What do you think?